app/views/finders/_results.mustache

```diff
@@ -1,3 +1,8 @@
+<div class="feed">
+  <span>Get updates to this list</span>
+  <a href="{{atom_url}}">feed</a>
```

Comment:
We should only use visually hidden text as a last resort as it is so often forgotten about when refactoring. If you are concerned that the link needs more text for users of screen readers then you really should consider adding that text for visual users as well. Low skilled users often need the same extra clues that a screen reader users are helped by.